extracted/konan.serializer/src/org/jetbrains/kotlin/serialization/konan/SourceFileMap.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+package org.jetbrains.kotlin.serialization.konan
+
+import org.jetbrains.kotlin.descriptors.SourceFile
+
+private class DeserializedSourceFile(val path: String, val index: Int) : SourceFile {
+    override fun getName(): String? = path
```

Comment:
Is it name or path?

Reply:
Whatever SourceElement.containingFile returns.